src/compas_timber/connections/l_butt.py

```diff
@@ -125,7 +123,7 @@ def add_features(self):
         start_main, start_cross = None, None
 
         try:
-            main_cutting_plane = self.get_main_cutting_plane()
+            main_cutting_plane = self.get_main_cutting_plane()[0]
```

Comment:
I find this more explicit, saying "this isn't a list, it's a tuple and I don't care about the second element". but could also just be a style preference. your call.
```suggestion
            main_cutting_plane, _ = self.get_main_cutting_plane()
```